fem/tmop_tools.cpp

```diff
@@ -218,6 +218,73 @@ void ParAdvectorCGOper::Mult(const Vector &ind, Vector &di_dt) const
 }
 #endif
 
+#ifdef MFEM_USE_GSLIB
+void InterpolatorFP::SetInitialField(const Vector &init_nodes,
+                                     const Vector &init_field)
+{
+   nodes0 = init_nodes;
+   field0 = init_field;
+   Mesh *m = mesh;
+#ifdef MFEM_USE_MPI
+   if (pmesh) { m = pmesh; }
+#endif
+   m->SetNodes(nodes0);
+
+   const double rel_bbox_el = 0.1;
+   const double newton_tol  = 1.0e-12;
+   const int npts_at_once   = 256;
+
+   FiniteElementSpace *f = fes;
+#ifdef MFEM_USE_MPI
+   if (pmesh) {finder = new FindPointsGSLIB(pfes->GetComm());}
+   else       {finder = new FindPointsGSLIB();}
+   if (pfes)  {f = pfes;}
+#else
+   finder = new FindPointsGSLIB();
+#endif
+   finder->Setup(*m, rel_bbox_el, newton_tol, npts_at_once);
+
+   field0_gf.SetSpace(f);
+   for (int i = 0; i < field0.Size(); i++)
+   {
+      field0_gf(i) = field0(i);
+   }
+
+   dim = f->GetFE(0)->GetDim();
+   const int pts_cnt = init_nodes.Size() / dim;
+   el_id_out.SetSize(pts_cnt);
+   code_out.SetSize(pts_cnt);
+   task_id_out.SetSize(pts_cnt);
+   pos_r_out.SetSize(pts_cnt*dim);
+   dist_p_out.SetSize(pts_cnt);
+}
+
+void InterpolatorFP::ComputeAtNewPosition(const Vector &new_nodes,
+                                          Vector &new_field)
+{
+
+   const int pts_cnt = new_nodes.Size() / dim;
+   Vector positionsToFind = new_nodes;
```

Comment:
Why do you need to copy this Vector (can't you just use new_field below)?

Reply:
I guess we need to change to `const` the first argument in `FindPoints.` Should be ok.